sdk/ai/azure-ai-generative/azure/ai/generative/synthetic/qa.py

```diff
@@ -204,20 +211,43 @@ def export_to_file(self, output_path: str, qa_type: QAType, results: Union[List,
         if not isinstance(results[0], List):
             results = [results]
         
-        for qs_and_as in results:
-            chat_history = []
-            for question, answer in qs_and_as: 
-                if qa_type == QAType.CONVERSATION:
-                    # Chat History columns:
-                    data_dict["chat_history"].append(json.dumps(chat_history))
-                    data_dict["chat_input"].append(question)
-                    chat_history.append({"inputs": {"chat_input": question}, "outputs": {"chat_output": answer}})
-                else:
-                    # QnA columns:
-                    data_dict["question"].append(question)   
+        if output_format == OutputStructure.PROMPTFLOW:
+            
+            if qa_type == QAType.CONVERSATION and not ("chat_history_key" in field_mapping and "question_key" in field_mapping and "answer_key" in field_mapping):
+                raise Exception("Field mapping for Promptflow output with Conversation must contain following keys: chat_history_key, question_key, answer_key")
+            elif not ("question_key" in field_mapping):
+                raise Exception(f"Field mapping for Promptflow output with {qa_type} must contain following keys: question_key")
 
-                data_dict["ground_truth"].append(answer)  # Consider generated answer as the ground truth
+            question_key = field_mapping["question_key"]
+            answer_key = field_mapping["answer_key"]
+            chat_history_key = field_mapping["chat_history_key"]
```

Comment:
this will throw error when QA is not conversation. and chat_history_key is not provided.

Reply:
Good point, will fix